modules/calendar/search.php

```diff
@@ -0,0 +1,173 @@
+<?php
+require_once("../../interface/globals.php");
+require_once("$srcdir/calendar.inc");
+require_once("$srcdir/patient.inc");
+require_once("$srcdir/appointments.inc.php");
+require_once("$srcdir/formatting.inc.php");
+
+$dateFormat = DateFormatRead();
+
+if ($_SESSION['authorizeduser'] == 1) {
+  $facilities = getFacilities();
+} else {
+  $facilities = getUserFacilities($_SESSION['authId']);
+}
+
+$providers = getProviderInfo();
+
+// used to filter based on keywords in the form
+function keywordFilter($arr) {
+  if($_POST['keyword'] != "") {
+    $keyword = $_POST['keyword'];
+    $fmlName = $arr['fname'] . " " . $arr['mname'] . " " . $arr['lname'];
+    $flName = $arr['fname'] . " " . $arr['lname'];
+    return preg_match("/$keyword/i", $fmlName) || preg_match("/$keyword/i", $flName) || $keyword == $arr['pc_pid'];
+  }
+  else return true;
+}
+?>
+<html>
+<head>
+  <link href="<?php echo $GLOBALS['standard_js_path'] ?>bootstrap-3-3-4/dist/css/bootstrap.min.css" rel="stylesheet" />
+  <link href="<?php echo $GLOBALS['standard_js_path'] ?>jquery-datetimepicker-2-5-4/build/jquery.datetimepicker.min.css" rel="stylesheet" />
+  <link href="css/search.css" rel="stylesheet" />
```

Comment:
Look at nileshprasad137  work on the header file which can be used to pull in these items.